cpp/tests/interop/arrow_utils.hpp

```diff
@@ -105,6 +105,22 @@ get_arrow_array(std::vector<std::string> const& data, std::vector<uint8_t> const
   return string_array;
 }
 
+template <typename T>
+std::enable_if_t<std::is_same_v<T, cudf::string_view>, std::shared_ptr<arrow::Array>>
+get_arrow_large_string_array(std::vector<std::string> const& data,
```

Comment:
```suggestion
std::shared_ptr<arrow::Array> get_arrow_large_string_array(std::vector<std::string> const& data,
```
The template is unnecessary here.